proto/minder/v1/minder.proto

```diff
@@ -2927,7 +3011,7 @@ message ListEvaluationHistoryRequest {
         (buf.validate.field).repeated = {
             items: {
                 string: {
-                    pattern: "[,-./[:word:]]*",
+                    pattern: "^[,-./[:word:]]*$",
```

Comment:
Ditto, why allow `,` here?

Reply:
Because of - https://github.com/mindersec/minder/blob/3872633d436f63fda7c3f0cb842258153946f6bb/internal/controlplane/handlers_evalstatus.go#L283-L287